object_test.go

```diff
@@ -729,6 +729,10 @@ func TestObject_ContainsValue(t *testing.T) {
 		value.NotContainsValue("XXX")
 		value.chain.assertNotFailed(t)
 		value.chain.clearFailed()
+
+		value.ContainsValue(make(chan int))
+		value.chain.assertFailed(t)
+		value.chain.clearFailed()
```

Comment:
Let's extract this into subtest "invalid argument" and move it to the end of the test.

Let's also add the similar check for NotContainsValue to the same subtest.

Reply:
added this new subtest and the Not containsValue check